lib/awful/tag.lua

```diff
@@ -833,11 +826,15 @@ function tag.setproperty(_tag, prop, value, ignore_setters)
     end
 end
 
---TODO move to client
-
 --- Tag a client with the set of current tags.
+-- @deprecated awful.tag.withcurrent
 -- @param c The client to tag.
 function tag.withcurrent(c)
+--     util.deprecate("c:to_selected_tags() instead of awful.tag.selectedlist")
+
+    -- It can't use c:to_selected_tags() because awful.tag is loaded before
+    -- awful.client
```

Comment:
I don't get this, `tag.withcurrent` will only be called long after all of `awful` is loaded, I think. No?